### PR TITLE
RHDEVDOCS-4070 Pipelines 1.7.1 Release Notes

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.10`, `enterprise-4.11`
- **JIRA issues**: [RHDEVDOCS-4070 RN and Errata for Pipelines 1.7.1](https://issues.redhat.com/browse/RHDEVDOCS-4070)
- **Preview pages**: Due to issues with Netlify, preview pages are not getting generated. Download and see this [locally generated HTML](https://drive.google.com/file/d/1oOrp9E5o8_MmrwXfwHfGQobx1AJPbUrb/view?usp=sharing) page on your browser. Look for the section "**Release notes for Red Hat OpenShift Pipelines General Availability 1.7.1**".
- **SME review**: @concaf @pradeepitm12      
- **QE review**: @ppitonak    
- **Peer-review**: @jc-berger 
- [**Engineering draft**](https://docs.google.com/document/d/1uab5nyKnyCGKxsfS_NRmauSE6rSsWHWh0sA7XZb1g9A/edit#heading=h.b47brzfzy6lx)

### CODE REVIEW

File: modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc [hunk 1]
```diff
@@ -26,7 +26,7 @@ a|
 a| 
 * If you continue using an earlier version of a cluster task, you might miss the latest features and critical security updates.
 * The earlier versions of cluster tasks that are not operational consume cluster resources.  
-* When upgraded , the Operator cannot manage the earlier VCT. You can delete the earlier VCT manually using the `oc delete clustertask` command, but you cannot restore it.  
+* When upgraded, the Operator cannot manage the earlier VCT. You can delete the earlier VCT manually using the `oc delete clustertask` command, but you cannot restore it.
```
Comment (sounix000) [inline modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc:29]: To reviewers: Resolved the comment https://github.com/openshift/openshift-docs/pull/45917#discussion_r889026309. This has no relation with release notes.

File: modules/op-release-notes-1-7.adoc [hunk 2]
```diff
@@ -339,4 +339,24 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
...
+// https://issues.redhat.com/browse/SRVKP-2243
+
+* Before this update, attempts to access resource URLs directly from the {tekton-hub} web console resulted in an Nginx `404` error. With this update, the {tekton-hub} web console image is fixed to allow accessing resource URLs directly from the {tekton-hub} web console.
+// https://issues.redhat.com/browse/SRVKP-2196
+
+* Before this update, the resource pruner job was designed in such a way that for each namespace, the {pipelines-title} Operator created command that ran in a separate container. With this update, all namespace based configuration are passed to the pruner job together, which results in execution of all the commands in a loop within one container.
```
Comment (pradeepitm12) [inline modules/op-release-notes-1-7.adoc:360]: Should we make it a little bit more simpler.

`
Before this update, the resource pruner job created n containers for n namespaces to prune resources, now the prune commands runs as loop in one container.
`

File: modules/op-release-notes-1-7.adoc [hunk 3]
```diff
@@ -339,4 +339,24 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
 // https://github.com/tektoncd/operator/pull/599
 
 
+[id="release-notes-1-7-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.7.1
+
+[id="fixed-issues-1-7-1_{context}"]
+=== Fixed issues
+
+* Before this update, upgrading the {pipelines-title} Operator deleted the data in the database associated with {tekton-hub}, and installed a new database. With this update, an Operator upgrade preserves the data.
```
Comment (jc-berger) [inline modules/op-release-notes-1-7.adoc:348]: you don't need the comma before `and`:

`...deleted the data in the database associated with {prod} and installed a new database.`

File: modules/op-release-notes-1-7.adoc (hunk 2 above)
Comment (jc-berger) [reply modules/op-release-notes-1-7.adoc]: I like what @pradeepitm12 suggested, but I don't know what the `n` stands for...number? To be consistent with the phrasing of the other notes, I would restructure Pradeep wrote, something like this:

`Before this update, the resource pruner job created containers for namespaces to prune resources. With this update, the prune commands run as a loop in one container.`

File: modules/op-release-notes-1-7.adoc [hunk 4]
```diff
@@ -339,4 +339,24 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
...
+// https://issues.redhat.com/browse/SRVKP-2243
+
+* Before this update, attempts to access resource URLs directly from the {tekton-hub} web console resulted in an Nginx `404` error. With this update, the {tekton-hub} web console image is fixed to allow accessing resource URLs directly from the {tekton-hub} web console.
+// https://issues.redhat.com/browse/SRVKP-2196
+
+Before this update, for each namespace the resource pruner job created a separate container to prune resources. With this update, the resource pruner job runs commands runs as loop for all namespaces in one container.
```
Comment (jc-berger) [inline modules/op-release-notes-1-7.adoc:360]: Last sentence, I don't think the verb tenses agree with one another, does this sound right?

`With this update, the resource pruner job runs command runs as a loop for all namespaces in one container.`

**Review by jc-berger — COMMENTED** [review]
the openshift checklist was consulted when reviewing this PR: https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.mgz8v4w1kg86

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

**Review by concaf — APPROVED** [review]
/lgtm

**Review by ppitonak — APPROVED** [review]
/lgtm

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Added some comments

**Review by abrennan89 — APPROVED** [review]

**Comment by abrennan89** [comment]
@sounix000 this needs a manual cherrypick to 4.9 since the automatic CP failed

**Pull request merged**